frontend_app_file/src/component/FileComponent.jsx

```diff
@@ -149,6 +149,7 @@ export class FileComponent extends React.Component {
             isVideo={props.isVideo}
             downloadPdfPageUrl={props.downloadPdfPageUrl}
             downloadPdfFullUrl={props.downloadPdfFullUrl}
+            previewUrls={props.previewUrls}
```

Comment:
```suggestion
            previewList={props.previewList}
```

Reply:
Add on propTypes and defaultProps and maybe change the `previewUrl` name too because now it's and obj and not just an url